cpp/src/io/parquet/experimental/hybrid_scan_helpers.cpp

```diff
@@ -231,26 +247,8 @@ std::vector<std::vector<cudf::size_type>> aggregate_reader_metadata::filter_row_
   std::optional<std::reference_wrapper<ast::expression const>> filter,
   rmm::cuda_stream_view stream) const
 {
-  // Return all row groups if no filter expression
```

Comment:
No need for this as we check the converted expression in `hybrid_scan_impl.cpp` everytime.